cli/command/stack/ps.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func newPsCommand(dockerCli command.Cli) *cobra.Command {
+func newPsCommand(dockerCli command.Cli, common *commonOptions) *cobra.Command {
```

Comment:
I don't think `common` needs to be a pointer here as it's always non-nil?

Reply:
Well, it's because newPsCommand is called during command instantiation, but the common.orchestrator value will be updated in the stack/cmd.go:PersistentPreRunE. That's why it is passed as a pointer and not as a copied value.